Readme.md

```diff
@@ -9,14 +9,16 @@ import Hasher from "fr32-sha2-256-trunc254-padded-binary-tree-multihash"
 
 
 export const digestStream = async (source: AsyncIterable<Uint8Array>) => {
+  // allocate buffer to hold the multihash
+  const digest = new Uint8Array(Hasher.MAX_SIZE)
+  
   const hasher = Hasher.create()
   for (const chunk of source) {
     hasher.write(chunk)
   }
 
-  const digest = new Uint8Array(Hasher.prefix.length + Hasher.size)
-  // Write digest
-  hasher.digestInto(
+  // Write digest and capture end offset
+  const offset = hasher.digestInto(
```

Comment:
Is it offset or length? Length seems more typical...

Reply:
It is an `endOffset` so if you passed offset (2nd param) `0` than it's same as length, but if you passed `n` it will be `n + length`. I don't mind changing it to `length`, but user most likely will have to do `n + length` themselves.

Reply:
Not strong feels but in my experience when writing into a buffer you usually get back the number of bytes written, not the end offset...